### ISSUE TITLE
Perl::Critic spelling checks fail when tidyall runs from a subdir

### ISSUE THREAD

**Issue by oalders** [houseabsolute/perl-code-tidyall#85]
Consider this case in a perlcriticrc file:

```
[Documentation::PodSpelling]
stop_words_file = .stopwords
```

```
olaf@ct6-test:~/go/src/github.maxmind.com/maxmind/mm_website.git/go$ tidyall -g --no-cache

[checked] t/lib/TestForGo/MM/MinFraudDaemon/2/ValidRequests.pm
perlcritic --profile /home/olaf/go/src/github.maxmind.com/maxmind/mm_website.git/.perlcriticrc --program-extensions .pl --program-extensions .psgi /tmp/Code-TidyAll-ebdk/t/lib/TestForGo/MM/MinFraudDaemon/2/ValidRequests.pm failed
    exited with 2 - output was:
Could not open ".stopwords": No such file or directory
```

Unlike Tidyall, Perl::Critic doesn't have a `$ROOT` variable.  So, spelling checks which are run from a directory other than `$ROOT` are bound to fail.

Something like this fixes *some* of the cases:

```perl
diff --git a/bin/tidyall b/bin/tidyall
index 70ba3a4..de72af4 100755
--- a/bin/tidyall
+++ b/bin/tidyall
@@ -126,6 +126,7 @@ elsif ( ( $all_files || $svn_files || $git_files ) ) {
     }
     elsif ($git_files) {
         require Code::TidyAll::Git::Util;
+        chdir $ct->root_dir || die 'cannot chdir to ' . $ct->root_dir;
         @paths = Code::TidyAll::Git::Util::git_modified_files( $ct->root_dir );
     }
 }
```

I don't know the code well enough to say whether it could just `chdir` to `$ROOT` by default before running a command.  I'm happy to provide a patch if you have any guidance for me.

**Comment by autarch** [comment]
That's definitely not the right place to put the `chdir` call, but I think it should be safe to put it _somewhere_. I think the right place to do this is somewhere in `Code::TidyAll` itself using `File::pushd`. I think the best place is probably the `process_paths` method.

**Comment by autarch** [comment]
Fixed in v0.69
